plutus-playground-lib/src/Playground/API.hs

```diff
@@ -167,25 +168,21 @@ toSimpleArgumentSchema schema@Schema {..} =
                     -- of deadlines we're going with the quick
                     -- solution: Duck typing. If a schema looks like a
                     -- `Value` type, then assume it is a `Value` type.
-                    let fields =
-                            HM.toList $
-                            extractReference <$> view properties schema
-                     in if fields == valueTypeFields
-                            then ValueSchema fields
-                            else SimpleObjectSchema fields
+                 ->
```

Comment:
ugh, this `->` on a new line?!

Reply:
I've moved the comment around, and now the hindent is a little more pleasing.